templates/case-studies/thank-you.html

```diff
@@ -0,0 +1,36 @@
+{% extends "ai/base_ai.html" %}
+
+{% block title %}Thank you | Case Studies{% endblock %}
+
+{% block head_extra %}<meta name="robots" content="noindex" />{% endblock %}
+
+{% block content %}
+  {% with thanks_context="contacting our team." %}
+    {% include "shared/_thank_you.html" %}
```

Comment:
this Thank You page is quite outdated, I think we need a new one here. @lyubomir-popov Do we have a sketch of the new version somewhere already?

Reply:
I think I will remove the thank you page and add a success banner instead, since Gloria asked for it to be a popup modal and there was no explicit request for a thank you page.